Client/src/components/categoryRow.tsx

```diff
@@ -0,0 +1,36 @@
+import MovieTile from "./movieTile"
+type CategoryRowPorps = {
+    title:string,
+    moviesList:FilmListType
```

Comment:
Nie ma żadnego error z inportowaniem type FilmListType?

Reply:
bo moim zdaniem to trzeba zaimportować z pliku

Reply:
Wiem, że powinno się importować i w kolejnych commitach typy były importowane, lecz w tym przypadku kod ani aplikacja nie wykazywała żadnego błędu, VSC sam stworzył alias który prowadził do typu.
Zostało to zmienione w branchu moviePage ponieważ tam wykorzystywany jest inny typ danych.